<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTab</class>
 <widget class="QWidget" name="MainTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="KMessageWidget" name="diveNotesMessage"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="notesTab">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>523</width>
            <height>739</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>8</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="2" column="3">
              <widget class="QLabel" name="durationLabel">
               <property name="text">
                <string>Duration</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="dateLabel">
               <property name="text">
                <string>Date</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="timeLabel">
               <property name="text">
                <string>Time</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="airTempLabel">
               <property name="text">
                <string>Air temp.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="waterTempLabel">
               <property name="text">
                <string>Water temp.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QDateEdit" name="dateEdit">
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
               <property name="timeSpec">
                <enum>Qt::UTC</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QTimeEdit" name="timeEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="timeSpec">
                <enum>Qt::UTC</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="airtemp">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="watertemp">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="depthLabel">
               <property name="text">
                <string>Depth</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="depth"/>
             </item>
             <item row="3" column="3">
              <widget class="QLineEdit" name="duration"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="LocationLayout" stretch="0,1">
               <item>
                <widget class="QLabel" name="LocationLabel">
                 <property name="text">
                  <string>Location</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="locationTags">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="textFormat">
                  <enum>Qt::RichText</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>2</number>
               </property>
               <item>
                <widget class="DiveLocationLineEdit" name="location"/>
               </item>
               <item>
                <widget class="QToolButton" name="editDiveSiteButton">
                 <property name="toolTip">
                  <string>Edit dive site</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="../../subsurface.qrc">
                   <normaloff>:geocode</normaloff>:geocode</iconset>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QtWaitingSpinner" name="waitingSpinner" native="true"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLineEdit" name="diveTripLocation"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="DivemasterLabel">
               <property name="text">
                <string>Divemaster</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="BuddyLabel">
               <property name="text">
                <string>Buddy</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="TagWidget" name="divemaster">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="TagWidget" name="buddy">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,0,1">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="RatingLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Rating</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="visibilityLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Visibility</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="SuitLabel">
               <property name="text">
                <string>Suit</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignVCenter">
              <widget class="StarWidget" name="rating" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="1" alignment="Qt::AlignVCenter">
              <widget class="StarWidget" name="visibility" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="suit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_9">
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <item row="1" column="1">
              <widget class="QComboBox" name="DiveType"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="TagLabel">
               <property name="text">
                <string>Tags</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="TypeLabel">
               <property name="text">
                <string>Dive mode</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="TagWidget" name="tagWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="lineWrapMode">
                <enum>QPlainTextEdit::NoWrap</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="NotesLabel">
               <property name="text">
                <string>Notes</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="notesAndSocialNetworksLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextEdit" name="notes">
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="socialNetworks" native="true">
                 <layout class="QVBoxLayout" name="socialNetworksLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="equipmentTab">
      <attribute name="title">
       <string>Equipment</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>523</width>
            <height>739</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="equipmentTabScrollAreaLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>2</number>
           </property>
           <item row="1" column="0">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="cylinderWeightsLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QSplitter" name="splitter">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <widget class="TableView" name="cylinders" native="true"/>
                <widget class="TableView" name="weights" native="true"/>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>desktop-widgets/kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StarWidget</class>
   <extends>QWidget</extends>
   <header>desktop-widgets/starwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QWidget</extends>
   <header>desktop-widgets/tableview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagWidget</class>
   <extends>QPlainTextEdit</extends>
   <header>desktop-widgets/tagwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QtWaitingSpinner</class>
   <extends>QWidget</extends>
   <header>desktop-widgets/qtwaitingspinner.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DiveLocationLineEdit</class>
   <extends>QLineEdit</extends>
   <header>desktop-widgets/locationinformation.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../subsurface.qrc"/>
 </resources>
 <connections/>
</ui>
